apps/client/ui/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,19 @@
+import type { FC } from 'react'
+
+type AvatarProps = {
```

Comment:
I saw you stopped using `ComponentProps` type from `react` in some components - is it because you decided it is not needed or you just forgot? 😄

Reply:
I don't know if I am using it correctly - when I had a Button component it was easy because I was extending the ComponentPropsWithoutRef<'button'> but what should I use for example for the Avatar component? ComponentPropsWithoutRef<'div'>? I wasn't sure so I decided not to use it at all 😅